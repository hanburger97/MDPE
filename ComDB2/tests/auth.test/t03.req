set user user2
set password password2
select 'user2'
#only op can turn off authentication
put authentication off
