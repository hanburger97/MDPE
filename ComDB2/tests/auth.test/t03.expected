('user2'='user2')
[put authentication off] failed with rc -3 User does not have OP credentials
